lib/_http_client.js

```diff
@@ -311,7 +311,7 @@ ClientRequest.prototype._implicitHeader = function _implicitHeader() {
 
 ClientRequest.prototype.abort = function abort() {
   if (!this.aborted) {
-    process.nextTick(emitAbortNT.bind(this));
+    process.nextTick(emitAbortNT, this);
```

Comment:
Unless I'm missing something, these two lines are not equivalent. The existing one calls `emitAbortNT()` with `this` set to the current value of `this`. The new code calls `emitAbortNT()` with `this` as the first argument.

Reply:
Oops, I *am* missing something, namely the changes to `emitAbortNT()` below.  I'll delete this comment and my previous one.